crates/test-framework/src/spicetest/vector_search/worker.rs

```diff
@@ -120,13 +128,23 @@ pub(crate) struct VectorSearchWorker {
 }
 
 #[derive(Deserialize)]
-pub(crate) struct Match {
-    score: f64,
+#[allow(dead_code)]
```

Comment:
Currently using `dead_code` as usage/functionality to calculate score will be added next